src/EnergyPlus/UnitarySystem.hh

```diff
@@ -63,6 +63,69 @@ struct EnergyPlusData;
+        std::string control_type;
+        std::string controlling_zone_or_thermostat_location;
+        std::string dehumidification_control_type;
+        std::string availability_schedule_name;
+        std::string air_inlet_node_name;
+        std::string air_outlet_node_name;
+        std::string supply_fan_object_type;
+        std::string supply_fan_name;
+        std::string fan_placement;
+        std::string supply_air_fan_operating_mode_schedule_name;
+        std::string heating_coil_object_type;
+        std::string heating_coil_name;
+        Real64 dx_heating_coil_sizing_ratio;
+        std::string cooling_coil_object_type;
+        std::string cooling_coil_name;
+        std::string use_doas_dx_cooling_coil;
+        Real64 minimum_supply_air_temperature;
+        std::string latent_load_control;
+        std::string supplemental_heating_coil_object_type;
+        std::string supplemental_heating_coil_name;
+        std::string cooling_supply_air_flow_rate_method;
+        Real64 cooling_supply_air_flow_rate;
+        Real64 cooling_supply_air_flow_rate_per_floor_area;
+        Real64 cooling_fraction_of_autosized_cooling_supply_air_flow_rate;
+        Real64 cooling_supply_air_flow_rate_per_unit_of_capacity;
+        std::string heating_supply_air_flow_rate_method;
+        Real64 heating_supply_air_flow_rate;
+        Real64 heating_supply_air_flow_rate_per_floor_area;
+        Real64 heating_fraction_of_autosized_heating_supply_air_flow_rate;
+        Real64 heating_supply_air_flow_rate_per_unit_of_capacity;
+        std::string no_load_supply_air_flow_rate_method;
+        Real64 no_load_supply_air_flow_rate;
+        Real64 no_load_supply_air_flow_rate_per_floor_area;
+        Real64 no_load_fraction_of_autosized_cooling_supply_air_flow_rate;
+        Real64 no_load_fraction_of_autosized_heating_supply_air_flow_rate;
+        Real64 no_load_supply_air_flow_rate_per_unit_of_capacity_during_cooling_operation;
+        Real64 no_load_supply_air_flow_rate_per_unit_of_capacity_during_heating_operation;
+        Real64 maximum_supply_air_temperature;
+        Real64 maximum_outdoor_dry_bulb_temperature_for_supplemental_heater_operation;
+        std::string outdoor_dry_bulb_temperature_sensor_node_name;
+        Real64 maximum_cycling_rate;
+        Real64 heat_pump_time_constant;
+        Real64 fraction_of_on_cycle_power_use;
+        Real64 heat_pump_fan_delay_time;
+        Real64 ancillary_on_cycle_electric_power;
+        Real64 ancillary_off_cycle_electric_power;
+        Real64 design_heat_recovery_water_flow_rate;
+        Real64 maximum_temperature_for_heat_recovery;
+        std::string heat_recovery_water_inlet_node_name;
+        std::string heat_recovery_water_outlet_node_name;
+        std::string design_specification_multispeed_object_type;
+        std::string design_specification_multispeed_object_name;
+
+        UnitarySysInputSpec();
+
+        ~UnitarySysInputSpec()
+        {
+        }
+    };
+
```

Comment:
new struct to hold UnitarySystem model inputs. This array will be used to translate inputs for other object types to comparable UnitarySystem inputs.

Reply:
OK, so this is essentially just a list of the input fields that unitary uses.  I like having a standardized signature that other parent objects can use as we adapt them.  This is really good.  For the record you don't need to explicitly list out the empty constructor and destructors for classes like this that won't be involved in inheritance and the like.